test/e2e/framework/helpers/yunikorn/wrappers.go

```diff
@@ -131,8 +131,7 @@ func RestoreConfigMapWrapper(oldConfigMap *v1.ConfigMap, annotation string) {
 	var e, err3 = k.UpdateConfigMap(c, configmanager.YuniKornTestConfig.YkNamespace)
 	Ω(err3).NotTo(HaveOccurred())
 	Ω(e).NotTo(BeNil())
-	// Updating scheduler pod annotation to trigger force refresh of configmaps
-	Ω(k.RemoveYunikornSchedulerPodAnnotation(annotation)).NotTo(HaveOccurred())
```

Comment:
It looks like `KubeCtl.RemoveYunikornSchedulerPodAnnotation` is only called here. We can also remove it.

https://github.com/apache/yunikorn-k8shim/blob/292e26fd9d910be39bb98f3bc01b7445179dff9e/test/e2e/framework/helpers/k8s/k8s_utils.go#L945

Reply:
Thanks @FrankYang0529 I will update in next rev.

Reply:
Done in the new PR rev submitted, thanks.